js/genkit/src/genkit.ts

```diff
@@ -571,23 +555,21 @@ export class Genkit {
     },
     runner: EmbedderFn<ConfigSchema>
   ): EmbedderAction<ConfigSchema> {
-    return runWithRegistry(this.registry, () =>
-      defineEmbedder(options, runner)
-    );
+    return defineEmbedder(this.registry, options, runner);
   }
 
   /**
    * create a handlebards helper (https://handlebarsjs.com/guide/block-helpers.html) to be used in dotpormpt templates.
```

Comment:
Keep these great typos

Reply:
lol didn't even notice! Keeping for sure!